openlcs/libs/corgi.py

```diff
@@ -81,6 +85,23 @@ def get_api_endpoint(self):
             return os.getenv("CORGI_API_PROD")
         return os.getenv("CORGI_API_STAGE")
 
+    @staticmethod
+    def get_corgi_access_token():
+        return f"Token {os.getenv('CORGI_ACCESS_TOKEN')}"
+
+    def corgi_request(self, path, access_token, method='GET', **kwargs):
```

Comment:
It'll be great to document under what scenario this should be used.

I'm asking because I assume you put it here for a reason with default method `GET`, while I don't see from corgi API any restrictions for GET requests at all.